source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -216,6 +216,10 @@ When you run this code, your output should look something like this:
 
    [Flower [id=5f7f87659ed5b07cf3480a06, name=rose, colors=[green, red], isBlooming=false, height=25.4]]
 
+
+To learn more about the read and write operations you can perform, see
+:ref:`java-read-operations` and :ref:`java-write-operations`. 
```

Comment:
Issue: the paragraph interrupts the flow to the note by switching the user to a different context. The note is related to the previous step, not to learning more about read and write operations.